web/directory/management/commands/parse_coop_csv.py

```diff
@@ -139,17 +171,11 @@ def get_address_pks(file_path, city_pks):
                 # If there are no lat or lon coords provided, attempt to figure
                 # them out
                 if not lat or not lon:
-                    svc = LocationService()
-                    ret = svc.get_coords(
-                        address=street,
-                        city=city,
-                        zip=postal_code,
-                        state_code=state_id,
-                        country_code="US"
-                    )
-                    if ret:
-                        lat = ret[0]
-                        lon = ret[1]
+                    # use new class object getGeoLoc to get geo info
```

Comment:
I think if we moved the logic from the class above into the service, we could leave lines 142 - 152 exactly as they are and you won't have to add lines 175 -178.